Services/Mail/classes/AutoResponder/class.ilAutoResponderDatabaseRepository.php

```diff
@@ -0,0 +1,115 @@
+<?php
+declare(strict_types=1);
+
+/**
```

Comment:
The license header is wrong. I know that `PhpStorm` automatically formats it, but this is wrong.